.eslintrc.js

```diff
@@ -5,28 +5,23 @@ module.exports = {
     commonjs: true,
     es6: true
   },
-  extends: [
-    "eslint:recommended",
-    "plugin:react/recommended",
-    "plugin:jest/recommended",
-    "plugin:prettier/recommended",
-    "prettier/react"
-  ],
-  parser: "babel-eslint",
+  "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended", "prettier"],
```

Comment:
```suggestion
  "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended", "plugin:prettier/recommended"],
```

After this change, the eslint started to show all the prettier errors in the editor.

Reply:
thanks! updated